File: CSharp/LastCharacter.cs
namespace CSharp
{
    // Create a function which makes the last character of a string repeat n number of times.
    // https://edabit.com/challenge/ZCAugLuYPbzcC4Eos
    public static class LastCharacter
    {
        public static string ModifyLast(string str, int n) => $"{str}{new string(str[str.Length - 1], n - 1)}";
    }
}

File: js/lastCharacter.js
// Create a function which makes the last character of a string repeat n number of times.
// https://edabit.com/challenge/iKEQo67KofMY4NSsP

const modifyLast = (str, n) => `${str}${str.slice(-1).repeat(n - 1)}`;

module.exports = modifyLast;
